src/System.Runtime.Intrinsics/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
Given that the implementation is in x86/x64 CoreLib only, I suspect that this is going to need two build configurations: netcoreapp-x86/x64 and netcoreapp. (Or maybe even 3 build configurations - I am not sure whether x86/x64 can be together.)